Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -291,6 +291,11 @@ def audit_file
 
     problem "File should end with a newline" unless text.trailing_newline?
 
+    if !Dir.glob("#{formula.path.to_s.gsub(/.rb$/, "")}@*.rb").empty? &&
+       formula.aliases.grep(/.@\d/).empty?
+      problem "Formula has other versions so should have an alias to the current version."
```

Comment:
Can you print exactly what that alias should be (or at least a suggestion) so it's a bit easier for people to act on?